lib/Core/Executor.h

```diff
@@ -430,28 +409,34 @@ class Executor : public Interpreter {
   const InstructionInfo & getLastNonKleeInternalInstruction(const ExecutionState &state,
       llvm::Instruction** lastInstruction);
 
-  bool shouldExitOn(enum TerminateReason termReason);
-
-  // remove state from queue and delete
+  /// Remove state from queue and delete state
   void terminateState(ExecutionState &state);
-  // call exit handler and terminate state
-  void terminateStateEarly(ExecutionState &state, const llvm::Twine &message);
-  // call exit handler and terminate state
+
+  /// Call exit handler and terminate state normally
+  /// (end of execution path)
   void terminateStateOnExit(ExecutionState &state);
-  // call error handler and terminate state
+  /// Call exit handler and terminate state early
+  /// (e.g. due to state merging or memory pressure)
+  void terminateStateEarly(ExecutionState &state, const llvm::Twine &message,
+                           StateTerminationType terminationType);
+  /// Call error handler and terminate state in case of program errors
```

Comment:
Can you leave empty lines b/w function declarations?

Reply:
You mean I should add some? There are none: https://github.com/klee/klee/blob/a38ee31b70101955f7360f1e077261daf43c3378/lib/Core/Executor.h#L435

Reply:
Yes, add one empty line between function declarations.